packages/core/src/components/search/Filter/FilterFixed.tsx

```diff
@@ -0,0 +1,109 @@
+import { setFacet, toggleFacet, useSearch } from '@faststore/sdk'
+
+import {
+  FacetBoolean as UIFacetBoolean,
+  FacetBooleanItem as UIFacetBooleanItem,
+  FacetRange as UIFacetRange,
+  Facets as UIFacets,
+  Filter as UIFilter,
+} from '@faststore/ui'
+import type { Filter_FacetsFragment } from '@generated/graphql'
+import { useFormattedPrice } from 'src/sdk/product/useFormattedPrice'
+import { useFilter } from './useFilter'
+
+interface Props {
+  /**
+   * The array that represents the details of every facet.
+   */
+  facets: Filter_FacetsFragment[]
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Title for the `FilterFixed` component.
+   */
+  title?: string
+}
+
+function FilterFixed({
```

Comment:
Can we rename this component? hahaha I don't have a better idea for it, but I think I think `FliterDesktop` sounds better. 🙈

Reply:
Nice! I was with that doubt as well 😉  hahah